### PR TITLE
Update kubevirt to v0.19.0

### PR DESCRIPTION
**Pull request by slintes**
This PR updates KubeVirt code to v0.19.0 and adds the new kubevirt.io/client-go dependency.

~~WIP! We don't have the client-go code on github.com/kubevirt yet, so I used my own fork for this.
Please check if everything works as expected while I get the code into the kubevirt repo.
Using `source` for pointing to `kubevirt.io/kubevirt/staging/src/kubevirt.io/client-go` (where the code lives) didn't work.~~

@rthallisey 

/hold

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -8,7 +8,7 @@ OPERATOR_IMAGE     ?= kubevirt/hyperconverged-cluster-operator
 REGISTRY_NAMESPACE ?=
 
 build: $(SOURCES) ## Build binary from source
-	go build -i -ldflags="-s -w" -o _out/hyperconverged-cluster-operator ./cmd/manager > /dev/null
+	go build -i -ldflags="-s -w" -o _out/hyperconverged-cluster-operator ./cmd/manager
```
Comment (slintes) [inline Makefile]: ups, didn't want to commit this. But, why do you do this?

Comment (rthallisey) [reply Makefile]: probably shouldn't be there since it's suppressing stdout.

**Comment by rthallisey** [comment]
ci test please

**Comment by rthallisey** [comment]
ci test please

**Comment by rthallisey** [comment]
ci test please

File: Makefile (hunk 1 above)
Comment (djzager) [reply Makefile]: I had this eerie suspicion that this was something I had done...but I have no answer to your question why this was done.

**Comment by rthallisey** [comment]
ci test please

File: Makefile (hunk 1 above)
Comment (slintes) [reply Makefile]: ack, then I will leave it :+1:

**Comment by slintes** [comment]
Updated to kubevirt's client-go repository.
Anything else needed for this?

**Comment by rthallisey** [comment]
I don't think anything else is needed @slintes.  The CI keeps failing on something unrelated.

**Comment by rthallisey** [comment]
This has passed on Openshift and K8s on different CI runs without changes.  Going to merge since the failure is unrelated and it blocks other work.  We'll handle CI in another PR

**Pull request merged**